stable/influxdb/templates/service.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+  {{- if .Values.config.http.enabled }}
+  - name: api
+    port: {{ .Values.config.http.bind_address }}
```

Comment:
@lachie83 they are specified as `bind_address` in the telegraf config. I've maintained that syntax here. Because of the way I generate the config I'm bound to the telegraf naming. Don't know if I can change this without significant changes to the chart.